Update connector documentation (#5723)
opensearch-project/documentation-website: _ml-commons-plugin/remote-models/index.md

### Node settings

> Models based on external connectors consume fewer resources. Therefore, you can deploy any model from a standalone connector using data nodes. To make sure that your standalone connection uses data nodes, set `plugins.ml_commons.only_run_on_ml_node` to `false`:

Author: ```suggestion
Externally hosted models that are deployed using connectors consume fewer resources. Therefore, you can deploy such models model on data nodes. To make sure that your model connection uses data nodes, set `plugins.ml_commons.only_run_on_ml_node` to `false`:
```

> Take note of the returned `model_id` because you’ll need it to deploy the model.

Reviewer: Should we highlight this key info in a callout?

Author: I don't use a callout in other places where there are similar steps... I think it's because of the proximity of the next step, which requires the model ID. Users can also retrieve the model ID after the fact if they don't note it immediately.
